src/main/webapp/style.css

```diff
@@ -36,6 +36,9 @@
 a {
   color: black;
 }
+a:hover {
+  color: black;
+}
```

Comment:
Nit: I feel like I've seen Zoe comment on this before, but is this necessary? Doesn't color stay the same on hover by default?

Reply:
I changed it in that PR but then I realized that it was necessary after all, links by default are blue, and the hover by default is a different(?) blue, but overwriting the link colour does not overwrite the hover colour, so I'm adding it back.

Reply:
Got it, thanks for clarifying